docs/integrations/ember-and-json-api.md

```diff
@@ -102,18 +118,16 @@ export default DS.JSONAPISerializer.extend({
 
 ```
 
+
 ## Including Nested Resources
 
-Previously, `store.find` and `store.findRecord` did not allow specification of any query params.
-The ActiveModelSerializers default for the `include` parameter is to be `nil` meaning that if any associations are defined in your serializer, only the `id` and `type` will be in the `relationships` structure of the JSON API response.
+Ember-data can request related records by using `include`.  To tell it to make that request, use the form below when requesting records.
```

Comment:
Ember Data is referred to as a "Ember Data" rather than "Ember-data", see: https://github.com/emberjs/data